### PR TITLE
fix/SOC-6257: Update news logo and its position

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less [hunk 1]
```diff
@@ -217,20 +217,15 @@
...
-      right: 0;
-      top: 0;
-    }
-
     .newsRibbon {
         background-image: url("@{images-path}/social/skin/Activity/ribbon-news.png");
         background-repeat: no-repeat;
         position: absolute;
         width: 85px;
         height: 87px;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less]: The image has changed, so the size must be updated IMO, height=3px and width=73px.

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less [hunk 2]
```diff
@@ -217,20 +217,15 @@
...
         left: auto;
-        right: -9px;
-        top: -9px;
+        right: -12px;
+        bottom: 26px;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less]: Here is what I see:
image
There is a little gap at the bottom, so the bottom property must be updated.

Also, the image still contains little borders on top-right and bottom-left corners, they must be deleted.

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less (hunk 1 above)
Comment (aelbenney) [reply platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less]: yesss!

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less (hunk 2 above)
Comment (aelbenney) [reply platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less]: I will fix the gap on the bottom and I will check with Chaima about the white spaces!

**Pull request merged**